config/locales/server.en.yml

```diff
@@ -2378,7 +2378,7 @@ en:
   email_log:
     no_user: "Can't find user with id %{user_id}"
     anonymous_user: "User is anonymous"
-    suspended_not_pm: "User is suspended, not a message"
+    suspended_not_pm: "User is suspended, not a PM"
```

Comment:
PM seems not used anymore. They are however appeared as `message` in any place. I didn't confirm that